urbansim/utils/tests/test_yamlio.py

```diff
@@ -146,3 +147,53 @@ def test_frame_to_yaml_safe():
                  'col2': {0: 'a', 1: 'b', 2: 'c'}}
     y = yaml.dump(d, default_flow_style=False)
     assert_dfs_equal(pd.DataFrame(yaml.load(y)), df)
+
+
+def test_orderered_dict():
```

Comment:
I was planning on using this for DCM updates. It's really just cosmetic, but I wanted a little more control over the look of the yaml.

One thing to note is that constructing  a pandas DataFrame with an ordered dict will allow for controlling the column order, but not necessarily the row/index order. However, I think in most cases this can be handled with some re-indexing from the dictionary keys:

```python
 pd.DataFrame.from_dict(my_odict, orient='index').reindex(my_odict.keys()).T
```